.github/CODEOWNERS

```
@@ -1,3 +1,5 @@
-* @cameronwaterman
+* @rajsite @atmgrifter00 @cameronwaterman
 
 /src/datasources/asset @CiprianAnton @kkerezsi
+/src/datasources/products @richie-ni @ranjgith-ni
+/src/datasources/results @richie-ni @Ahalya-ni
```

```suggestion
/src/datasources/results @richie-ni @ranjgith-ni
```

You could mark me and richie as owners for both products and results